src/components/feed/PostFeed.js

```diff
@@ -131,9 +131,22 @@ class PostFeed extends Component {
   }
 
   handleDateTime(date) {
-    const dateLocal = new Date(date);
-    const timeLocal = dateLocal.toLocaleTimeString();
-    return (String(dateLocal.toDateString()) + " " + timeLocal);
+    const d = new Date(date);
+    const createdDate = d.toDateString();
+    const timeLocal = d.toLocaleTimeString();
```

Comment:
Remove redundant variable, declared but never used.

Reply:
I will review that again !